gunicorn/arbiter.py

```diff
@@ -154,7 +155,8 @@ def start(self):
                 for fd in os.environ.pop('GUNICORN_FD').split(','):
                     fds.append(int(fd))
 
-            self.LISTENERS = sock.create_sockets(self.cfg, self.log, fds)
```

Comment:
this would break if fds are afaik given by systemd. In such case the fds can still be bound per workers. Maybe a None socket can be given to each workers in the case they are created?